starknet_py/utils/sync/sync_test.py

```diff
@@ -76,12 +76,12 @@ async def test_asynchronous_versions():
 
 def test_sync_versions():
     contract = Contract("1")
-    repository = contract.get_repository_sync()
-    function = repository.get_function_sync()
+    repository = contract.get_repository_sync()  # pyright: ignore
+    function = repository.get_function_sync()  # pyright: ignore
 
     assert function.call_sync() == 1
     assert function.get_name() == "function X"
-    assert contract.example_class_method_sync() == 2
+    assert contract.example_class_method_sync() == 2  # pyright: ignore
```

Comment:
Why are ignores needed here?

Reply:
Because pyright does not recognize _sync methods :/

Reply:
I'd consider including some explanations here and in other places, why pyright was disabled. This probably also applies to code already merged